cmp/LoadMask.js

```diff
@@ -15,20 +16,17 @@ import './LoadMask.scss';
 /**
  * Simple LoadMask.
  *
- * This Mask currently will occupy the entire viewport.
- * Localized masking will be provided by a future option.
- *
  * The mask can be explicitly shown, or reactively bound to a PromiseModel.
  */
 @hoistComponent()
 export class LoadMask extends Component {
 
     BACKGROUND = 'rgba(0,0,0, 0.25)';
 
-    static defaultProps = {
-        isDisplayed: false,
-        model: null,
-        inline: true
+    static propTypes = {
+        isDisplayed: PT.bool,
+        model: PT.object,
+        inline: PT.bool
```

Comment:
isDisplayed is certainly self evident, but I would add a comment on model and inline. Something like: "PromiseModel instance that will show this mask whenever pending" and whatever exactly inline is.... 😄